site-content/source/modules/ROOT/pages/download.adoc

```diff
@@ -124,6 +124,27 @@ https://www.apache.org/dyn/closer.lua/cassandra/3.0.30/apache-cassandra-3.0.30-b
 ==== The PGP keys used to sign releases are available in this https://downloads.apache.org/cassandra/KEYS[KEYS,window=_blank] file.
 ----------
 
+{empty} +
+
+[openblock, inline100 inline-top]
+----------
+
+[discrete]
+=== Latest Cassandra Java Driver
+[discrete]
+==== Apache 4.x Java driver for Cassandra
+
+[.btn.btn--alt]
+https://www.apache.org/dyn/closer.lua/cassandra/cassandra-java-driver/4.18.1/apache-cassandra-java-driver-4.18.1.tar.gz[4.18.1,window=blank]
+
+(https://downloads.apache.org/cassandra/cassandra-java-driver/4.18.1/apache-cassandra-java-driver-4.18.1.tar.gz.asc[pgp,window=blank], https://downloads.apache.org/cassandra/cassandra-java-driver/4.18.1/apache-cassandra-java-driver-4.18.1.tar.gz.sha256[sha256,window=blank], https://downloads.apache.org/cassandra/cassandra-java-driver/4.18.1/apache-cassandra-java-driver-4.18.1.tar.gz.sha512[sha512,window=blank]) +
+(https://www.apache.org/dyn/closer.lua/cassandra/cassandra-java-driver/4.18.1/apache-cassandra-java-driver-4.18.1-source.tar.gz[source,window=blank]: https://downloads.apache.org/cassandra/cassandra-java-driver/4.18.1/apache-cassandra-java-driver-4.18.1-source.tar.gz.asc[pgp,window=blank], https://downloads.apache.org/cassandra/cassandra-java-driver/4.18.1/apache-cassandra-java-driver-4.18.1-source.tar.gz.sha256[sha256,window=blank], https://downloads.apache.org/cassandra/cassandra-java-driver/4.18.1/apache-cassandra-java-driver-4.18.1-source.tar.gz.sha512[sha512,window=blank])
+
+Java drivers for Cassandra are also available on
+https://central.sonatype.com/artifact/org.apache.cassandra/java-driver-core[Maven Central,window=blank].
```

Comment:
fyi, we (now) publish originally to 
https://repository.apache.org/content/groups/public/org/apache/cassandra/java-driver-core/ 

no objection to listing central sonatype, but artefacts may take time after first publish to appear there, while repository.apache.org immediately has them…

Reply:
I took a look at Spark and they use similar reference to Maven Central as does the Java driver's Readme.

Link with Spark
Spark artifacts are [hosted in Maven Central](https://search.maven.org/search?q=g:org.apache.spark). You can add a Maven dependency with the following coordinates:

https://spark.apache.org/downloads.html

Reply:
WFM